setup-cluster.sh

```diff
@@ -26,23 +39,23 @@ cat << EOF | tee ${VAGRANT_CONFIG}
 \$node_count = ${NODE_COUNT}
 \$master_count = 1
 
-\$vm_memory = 1024
-\$vm_vcpus = 1
-\$vm_disk = 45
+\$vm_memory = 16192
```

Comment:
Multinode with 16G of memory is a bit much. OpenShift only requires a minimum 500Mb. Is there a reason to default it this high?